dev-infrastructure/configurations/mvp-svc-cluster.bicepparam

```diff
@@ -20,7 +20,7 @@ param maestroPostgresServerStorageSizeGB = 32
 param deployMaestroPostgres = false
 param maestroPostgresPrivate = false
 
-param deployCsInfra = false
+param deployCsInfra = true
```

Comment:
what's the reason for changing this?

Reply:
This enables the deployment of an Azure Database for PostgreSQL instance in the P1 environment, which is the motivation of all these changes. We want CS to use that service instead of a postgresql pod. Also when using that service we want to authenticate against the DB using Microsoft Entra tokens